R/backend-oracle.R

```diff
@@ -114,6 +114,15 @@ sql_query_wrap.Oracle <- function(con, from, name = unique_subquery_name(), ...)
   }
 }
 
+#' @export
+sql_query_save.Oracle <- function(con, sql, name, temporary = TRUE, ...) {
+  build_sql(
+    "CREATE ", if (temporary) sql("GLOBAL TEMPORARY "), "TABLE \n",
+    as.sql(name, con), " AS ", sql,
```

Comment:
```suggestion
    "CREATE ", if (temporary) sql("GLOBAL TEMPORARY "), "TABLE\n",
    as.sql(name, con), " AS\n", sql,
```

Reply:
It improves readability a bit so I changed it here and for the other backends 👍